app/src/main/res/values/strings.xml

```diff
@@ -34,8 +34,8 @@
     <string name="resume_scroll">Resume scroll</string>
     <string name="uploading_image">Uploading file</string>
     <string name="settings">Settings</string>
-    <string name="nuuls_upload_disclaimer">External hosting provided by https://nuuls.com/i, use at your own risk.</string>
-    <string name="nuuls_upload_title">Custom media upload</string>
+    <string name="external_upload_disclaimer">External hosting provided by %1$s, use at your own risk.</string>
```

Comment:
OK, I'll revert back to the old key (`nuuls_*`). 
For other locale translation, I think it would be fine to just replace `https://nuuls.com/i` with `%1$s` :thinking:  .